### ISSUE TITLE
[Registry] Can't upload to update extension

### ISSUE THREAD

**Issue by uniphonic** [adobe/brackets#11932]
Every time I try to upload an update to my extension in the registry it gives me the error:

An error occurred, but unfortunately I don't know how to say this nicely: EACCES, mkdir '/home/aboutweb/tmp/bracketsPackage_1151018-12803-7y3jpz'

I subsequently tried deleting my extension, and re-uploading it, and it still gave me this error message.

**Comment by berthermans** [comment]
Unfortunately I get the same error. Is this being looked at right now?

**Comment by abose** [comment]
tagging @nethip @ingorichter

**Comment by uniphonic** [comment]
No word on this? I just checked and it's still doing the same thing.

**Comment by abose** [comment]
We are tracking this will update soon.

**Comment by ingorichter** [comment]
I've restarted the registry. Could you please try again @uniphonic nad let us know if the issue is resolved for you?

**Comment by acbarbosa1964** [comment]
Dear friend,
I have tried to upload my extension just now and still not workink

**Comment by berthermans** [comment]
Sorry, but after 5 days it still outputs the following error:

An error occurred, but unfortunately I don't know how to say this nicely: EACCES, mkdir '/home/aboutweb/tmp/bracketsPackage_1151023-19196-13uuy7x'

**Comment by uniphonic** [comment]
I just tried again, and I'm still getting the same type of error:

An error occurred, but unfortunately I don't know how to say this nicely: EACCES, mkdir '/home/aboutweb/tmp/bracketsPackage_1151023-19196-xyxd4o'

**Comment by zhaopengme** [comment]
me too.
`An error occurred, but unfortunately I don't know how to say this nicely: EACCES, mkdir '/home/aboutweb/tmp/bracketsPackage_1151023-19196-4fstsf'`

**Comment by abose** [comment]
We tried restarting the machines, but that didn't seem to fix it. Brackets extensions are maintained by a third party due to the opensource nature of the project. 
We are in touch with abouttheweb guys for resolving this issue. will post an update soon.

**Comment by uniphonic** [comment]
I'm still experiencing the issue, and to be clear, I've never changed my GitHub username. I only mention that because I saw the other ticket said that, which is not the case for me.

**Comment by abose** [comment]
@ryanstewart @madanbn @nethip

**Comment by unsyllable** [comment]
I'm experiencing the exact same issue, I tried uploading the zip file after compressing through terminal but it does the same thing. Can anyone help?

**Comment by nethip** [comment]
Please bear with us. Due to the ongoing Thanksgiving holidays, our efforts at reviving the backend is taking longer than usual and we apologize for the inconvenience. Please wait while we get it fixed and try to put in a more scalable & robust solution.

**Comment by EJSG** [comment]
Hope you guys are all having a great Thanksgiving! I, too, am experiencing the same error:
`An error occurred, but unfortunately I don't know how to say this nicely: EACCES, mkdir '/home/aboutweb/tmp/bracketsPackage_1151029-19196-1vr4z18' `

**Comment by ryanstewart** [comment]
Hey all, can you give it a shot and try uploading them again? We've rebooted the sever in hopes of getting things squared away.

**Comment by uniphonic** [comment]
Hi Ryan,
Nope. Still not working. Got this error:
An error occurred, but unfortunately I don't know how to say this nicely: EACCES, mkdir '/home/aboutweb/tmp/bracketsPackage_1151030-3195-kdnaoi'

**Comment by jvmvik** [comment]
:+1:

**Comment by uniphonic** [comment]
I saw the thumbs up, so i thought maybe that meant it's working? I just tried again, but I'm still getting the same kind of error:
An error occurred, but unfortunately I don't know how to say this nicely: EACCES, mkdir '/home/aboutweb/tmp/bracketsPackage_115112-3195-9s81u8'

**Comment by abose** [comment]
Its fixed now. Closing.
Please reopen if any issues found.

**Comment by uniphonic** [comment]
While the main symptom which this issue describes is fixed, I noticed that at the same time this issue was fixed another issue appeared which might be related:

The download counter for extensions seem to be broken?

If you have the "Extensions Rating" extension installed it shows download counts for all items, but all the download counts seem to have frozen since this fix was made (everything that was uploaded since then has remained locked at 0 for many days, though I know it's been downloaded many times.. Should we open a separate issue for this, or can we keep it in this thread?

Thanks!
